Modules/ArcGIS/DSCResources/ArcGIS_Server/ArcGIS_Server.psm1

```diff
@@ -987,10 +1018,86 @@ function Get-SingleClusterModeOnServer
         Write-Verbose $res.Content
         $response = $res.Content | ConvertFrom-Json
         Check-ResponseStatus $response -Url $GetDeploymentUrl
-        $response    
+        $response
     }else {
         Write-Verbose "[WARNING] Response from $GetDeploymentUrl is NULL"
     }
 }
 
+function Get-AdminSettings
+{
+    [CmdletBinding()]
+    Param
+    (
+        [System.String]
+        $ServerUrl,
+
+        [System.String]
+        $SettingUrl,
+
+        [System.String]
+        $Token,
+
+        [System.String]
+        $Referer
+    )
+
+    $RequestUrl  = $ServerUrl.TrimEnd("/") + "/" + $SettingUrl.TrimStart("/")
```

Comment:
Can you please replace with the Invoke-ArcGISRequest utility method. Aside from this everything else looks great.